### PR TITLE
Adds StarterPlayer, StarterPlayerScripts, StarterCharacterScripts

### PR DESCRIPTION
**Pull request by Kampfkarren**

### CODE REVIEW

**Comment by Kampfkarren** [comment]
DNM, there's an issue where habitat:loadFromFs doesn't seem to respect this.

File: lib/instances/StarterCharacterScripts.lua [hunk 1]
```diff
@@ -0,0 +1,4 @@
+local BaseInstance = import("./BaseInstance")
+local StarterCharacterScripts = BaseInstance:extend("StarterCharacterScripts")
```
Comment (LPGhatguy) [inline lib/instances/StarterCharacterScripts.lua]: It's not obvious on the wiki, but `StarterCharacterScripts` actually extends `StarterPlayerScripts` for... some reason.

Here it is laid out in my local single-file wiki:
image

Comment (Kampfkarren) [reply lib/instances/StarterCharacterScripts.lua]: Uh...huh. Alright.

**Comment by Kampfkarren** [comment]
Fixed.

**Comment by LPGhatguy** [comment]
Thanks!

**Pull request merged**